src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -1,18 +1,23 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+
 import AspectRatioEnum from '@triniti/schemas/triniti/common/enums/AspectRatio';
-import DateTimePicker from '@triniti/cms/plugins/blocksmith/components/date-time-picker';
```

Comment:
@jalopez13 - please add `DatePicker` back, you are using it here: https://github.com/triniti/cms-js/pull/40/files#diff-4d29fd03a7c8ceffda7bc8dadd579750R169

Reply:
@arunkarnati 

Added DatePicker.